lib/kafka_ex/api_versions.ex

```diff
@@ -0,0 +1,24 @@
+defmodule KafkaEx.ApiVersions do
+
+  def api_versions_map(api_versions) do
+    api_versions
+      |> Enum.map(fn version -> {version.api_key, version} end)
+      |> Map.new
```

Comment:
This function isn't supported by elixir 1.1

Reply:
thanks, I'll fix that. Any idea what to replace it with ?

Reply:
Nevermind :)

Reply:
From the logs of the CI? it seems I fixed that problem.

Reply:
Another option

```elixir
Enum.into(api_versions, %{}, fn version ->  {version.api_key, version} end)
```

Reply:
thanks, I keep forgetting about `Enum.into` !